modules/aws/state_manager/variables.tf

```diff
@@ -0,0 +1,12 @@
+variable "project_name" {
+  type        = string
+  description = "This is used to label the components of the session storage"
+}
+
+# OPTIONAL VARIABLES
+
+variable "environment" {
+  type        = string
+  description = "This is used to label the components of the session storage with the AWS environment the bucket is hosted on"
+  default     = "test"
```

Comment:
@nitpick: do we want to default this to test? 

@suggestion: maybe we default this to main? or something like that (open to suggestions)

Reply:
I reckon dev would be a good choice.

Covers aspects of both following a 'develop' branch. While also the defactor environment a user would use.